crates/adapters/hyperliquid/src/execution/mod.rs

```diff
@@ -48,14 +50,49 @@ pub struct HyperliquidExecutionClient {
     core: ExecutionClientCore,
     config: HyperliquidExecClientConfig,
     http_client: HyperliquidHttpClient,
-    ws_client: Option<HyperliquidWebSocketClient>,
+    ws_client: HyperliquidWebSocketClient,
     started: bool,
     connected: bool,
     instruments_initialized: bool,
     pending_tasks: Mutex<Vec<JoinHandle<()>>>,
 }
 
 impl HyperliquidExecutionClient {
+    /// Returns a reference to the configuration.
+    pub fn config(&self) -> &HyperliquidExecClientConfig {
+        &self.config
+    }
+
+    /// Returns true if the client is configured for testnet.
+    pub fn is_testnet(&self) -> bool {
```

Comment:
I think it's nice to refer to `self.config` so its more obvious what is configurable and avoids the boilerplate of repeating the config in the client (fine for a follow up).

Reply:
Done, thank you.